assets/tests/components/map.test.tsx

```diff
@@ -3,7 +3,7 @@ import "@testing-library/jest-dom"
 import { LatLng } from "leaflet"
 import React, { MutableRefObject } from "react"
 import { act } from "react-dom/test-utils"
```

Comment:
Minor thing while you're already updating this file, but the React Testing Library docs recommend importing their `act` wrapper: https://testing-library.com/docs/react-testing-library/api/#act